run-tests.sh

```diff
@@ -31,6 +31,57 @@ dryrun() {
   fi
 }
 
+# Takes one argument, being service name
+getServiceVersion() {
+  local command
+  local response
+  local version
+  command="g3kubectl get configmap manifest-versions -o json | jq -r .data.json | jq -r .$1"
+  response=$(eval "$command")
+
+  # Get last item of delimited string using string operators:
+  #    https://www.linuxjournal.com/article/8919
+  version=${response##*:}
+  echo $version
+}
+
+# Takes 3 arguments:
+#   $1 service name
+#   $2 test tag to avoid until version is greater than next arg
+#   $3 versio of service where tests apply >=
+#
+# ex: runServiceTestsIfVersion "fence" "@multipartupload" "3.0.0"
+runServiceTestsIfVersion() {
+  local currentVersion
+  currentVersion=$(getServiceVersion $1)
```

Comment:
idk much about bash style conventions but should you make sure the arguments are provided?